src/libs/actions/IOU.js

```diff
@@ -2910,7 +2936,12 @@ function approveMoneyRequest(expenseReport) {
             statusNum: CONST.REPORT.STATUS.APPROVED,
         },
     };
-    const optimisticData = [optimisticIOUReportData, optimisticReportActionsData];
+    const optimisticNextStepsData = {
+        onyxMethod: Onyx.METHOD.SET,
+        key: `${ONYXKEYS.COLLECTION.NEXT_STEP}${expenseReport.reportID}`,
+        value: null,
+    };
```

Comment:
I think it might be safe to add it here too but approving reports is only on Collect policies which always have some next steps so this might be fine to leave as is

Reply:
I got you, but let's do ti safely.